src/potatoes/Plot.java

```diff
@@ -44,9 +44,15 @@ boolean inBounds(int posX, int posY) {
 		return false;
 	}
 
-	private void checkPotato(){
+	public void checkPotato(){
+		if (this.potato.getContext().getState().toString() ==  "Healthy State"){
+			changeColor(parent.color(79, 36, 5));
+		}
 		if (this.potato.getContext().getState().toString() ==  "Contaminated State"){
-			changeColor(parent.color(25, 0, 38));
+			changeColor(parent.color(249, 94, 4));
+		}
+		if (this.potato.getContext().getState().toString() ==  "Contagious State"){
+			changeColor(parent.color(237, 14, 2));
 		}
```

Comment:
Si, tu as raison (http://stackoverflow.com/questions/9169249/why-we-use-if-else-if-instead-of-multiple-if-block-if-the-body-is-a-return-stat --> voir la réponse validée).
Je vais le changer !